app/models/concerns/import_utils.rb

```diff
@@ -4,6 +4,16 @@
 module ImportUtils
   extend ActiveSupport::Concern
 
+  AUDIO_SAVED = "audio saved".freeze
```

Comment:
moved status definitions into this concern, which is shared by both `PodcastImport` and `EpisodeImport`